File: CSharp/Trivia Assignment backend/Models/AnswerModel.cs
namespace Trivia_Assignment_backend.Models
{
    public class AnswerModel
    {
        public Guid QuestionId { get; set; }
        public string CorrectAnswer { get; set; }
        public string GivenAnswer { get; set; }
        public bool WasAnswerCorrect { get; set; }

        public AnswerModel() { }
        public AnswerModel(Guid questionId, string givenAnswer, string correctAnswer = "", bool wasAnswerCorrect = false)
        {
            QuestionId = questionId;
            GivenAnswer = givenAnswer;

            CorrectAnswer = correctAnswer;
            WasAnswerCorrect = wasAnswerCorrect;
        }
    }
}

File: react/trivia_assignment_quad_solutions/app/AnswerModel.ts
class AnswerModel {
    private questionId: string;
    private givenAnswer: string;
    private correctAnswer: string;
    private wasAnswerCorrect: boolean;

    constructor(questionId: string, givenAnswer: string, correctAnswer = "", wasAnswerCorrect = false) {
        this.questionId = questionId;
        this.givenAnswer = givenAnswer;
        this.correctAnswer = correctAnswer;
        this.wasAnswerCorrect = wasAnswerCorrect;
    }
}